deepce.sh

```diff
@@ -1112,6 +1115,92 @@ exploitDockerSock() {
   # TODO: Tidy up command
 }
 
+exploitSysModule(){
+  caps=$(capsh --print)
+  if [[ ! $caps =~ "cap_sys_module" ]];then
```

Comment:
```suggestion
  if [ ! $caps =~ "cap_sys_module" ];then
```

Reply:
Double square brackets do not work in sh (might need to re-test this still works)